src/Lstm/index.js

```diff
@@ -9,145 +9,94 @@
 A LSTM Generator: Run inference mode for a pre-trained LSTM.
 */
 
-import * as dl from 'deeplearn';
-import sampleFromDistribution from './../utils/sample';
+import * as tf from '@tensorflow/tfjs';
 
-const regexCell = /cell_[0-9]|lstm_[0-9]/gi;
-const regexWeights = /weights|weight|kernel|kernels|w/gi;
-const regexFullyConnected = /softmax/gi;
+const DEFAULTS = {
+  seed: 'a',
```

Comment:
Does this ever get used? I think we should probably require a seed or else pick a random seed from the available alphabet of a given model.

Reply:
It does, in the generate function. But I agree, it should use a better approach than just this seed

Reply:
Great, this doesn't have to be fixed here and can live as a separate issue. I can file if that makes sense?

Reply:
perfect

Reply:
issue added #118